src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+    </Console>
+  </Appenders>
+  <Loggers>
+    <Logger name="com.bakdata" level="debug" additivity="false">
```

Comment:
This was info before. Is debug intended here?

Reply:
There was no test logging config before. Only production but that was never used

Reply:
Ok, I thought it is the replacement for `log4j.logger.com.bakdata=INFO`